Source/mpq/mpq_writer.hpp

```diff
@@ -14,14 +14,10 @@
 namespace devilution {
 class MpqWriter {
 public:
-	bool Open(const char *path);
-
-	bool Close(bool clearTables = true);
-
-	~MpqWriter()
-	{
-		Close();
-	}
+	MpqWriter(const char *path);
+	MpqWriter(MpqWriter &&other) = default;
+	MpqWriter &operator=(MpqWriter &&other) noexcept;

```

Comment:
Did you mean this?
```suggestion
	MpqWriter &operator=(MpqWriter &&other) = default;
```

Reply:
True. Now I'm unsure if I need it... fixed. 🙂